<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMenuPorcentages</class>
 <widget class="QDialog" name="DialogMenuPorcentages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Probabilidad de Platillos</string>
   </property>
  </widget>
  <widget class="QLabel" name="EntranceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Entrada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MealLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Plato Fuerte:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DessertLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Postre:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEntrance">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>43</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMeal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>43</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDessert">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>43</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfigButton">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>164</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Configurar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
